transcendence-app/src/user/user.entity.ts

```diff
@@ -1,19 +1,7 @@
-import { IsInt, IsPositive, ValidateNested } from 'class-validator';
-import { UserDto } from './dto/user.dto';
-import { Constants } from '../shared/enums/commonconsts.enum';
-
-export class UserEntity {
-  private static lastUserId = Constants.UserIdStart;
-
-  constructor(user: UserDto) {
-    this.Dto = user;
-    this.id = UserEntity.lastUserId++;
-  }
-
-  @IsPositive()
-  @IsInt()
-  id: number;
-
-  @ValidateNested()
-  Dto: UserDto;
+export interface UserEntity {
```

Comment:
Now that we have a DB model established, maybe we can update this to represent users table?

Reply:
Thank you! I've made the change.

I've also renamed the `uuid` column to `id` and the `createdat` to `created_at` and added a default value. Please let me know what you think.